documentation/projects/proposals/project_improvement/20230914-implementation_plan_project_automations.md

```diff
@@ -0,0 +1,262 @@
+# Implementation plan: Project automations
+
+**Author**: @dhruvkb
+
+## Reviewers
+
+- [ ] TBD
+- [ ] TBD
+
+## Project links
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/1931)
+- [Project Proposal](/projects/proposals/project_improvement/20230914-project_proposal_project_improvements.md)
+
+## Overview
+
+This is the implementation plan describing new automations for the project
+board. More information about the project board and existing automations can be
+found in our [documentation](/meta/project_boards/index.md).
+
+## Historical context
+
+The new project board was created as a successor to our original project boards,
+that were "classic" GitHub Projects.
+
+- [Openverse](https://github.com/orgs/WordPress/projects/3)
+- [Openverse PRs](https://github.com/orgs/WordPress/projects/8)
+
+```{note}
+These classic project boards have been closed. They are no longer updated, nor
+referred to in any discussions.
+```
+
+These boards were highly automated. A number of automations related to these
+boards were deleted ([PR](https://github.com/WordPress/openverse/pull/2338))
+when they were closed. Those automations can serve as inspiration for their
+reboots for the new project board.
+
+## To-do
+
+The complete work of automating our boards can be broadly classified under three
+categories.
+
+### General improvements to boards
+
+1. Normalise emoji usage
+
+   The backlog board uses an emoji for each column. None of the other boards do.
+   Personally I appreciate them and would like all boards to use them but
+   wouldn't mind dropping them from the backlog board for consistency's sake.
```

Comment:
I also like having the emojis! I think it serves as an easy visual indicator of what column one is looking at without having to parse the title itself.

Reply:
Recorded your opinion in support of emojis in 2f61873..